.env.sample

```diff
@@ -1,7 +1,19 @@
-APP_PORT=
-DB_HOST=
-DB_USERNAME=
-DB_PASSWORD=
-DB_NAME=
-
-JWT_SECRET=
+POSTGRES_DB= 
+POSTGRES_USER= 
+POSTGRES_PASSWORD= 
+POSTGRES_HOST_WRITE=
+POSTGRES_HOST_READ=
+POSTGRES_PORT= 
+TYPEORM_CONNECTION= 
```

Comment:
Where do you use the variable that start with 'TYPEORM'?

Reply:
We are not using this